exo-go/src/aws_helpers/terraform_helpers.go

```diff
@@ -0,0 +1,82 @@
+package awsHelper
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+// InitAccount prepares a blank AWS account to be used with Terraform
+func InitAccount(bucketName, tableName, region string) error {
+	config := aws.NewConfig().WithRegion(region)
+	session := session.Must(session.NewSession())
```

Comment:
Lets not use Must since that panics on error. Instead we should be bubbling the error up.